.github/workflows/verify-commits.yml

```diff
@@ -0,0 +1,28 @@
+name: verify-commits
```

Comment:
IIRC we can't use github actions in the openshift organization as it's using a legacy github subscription and doesn't have access to the plans that the operator-framework has. We'd need to add something like a Makefile target and a wrapper script that runs these git commands, and add a prow job that runs that makefile target.

Reply:
@njhale Can you open a PR against the upstream repository with these changes sans the github actions? It would be nice to have this kind of validation now that we've finally landed the latest downstream port.

Reply:
> IIRC we can't use github actions in the openshift organization as it's using a legacy github subscription and doesn't have access to the plans that the operator-framework has

Ah, great point. I can try for a prow job instead.



> Can you open a PR against the upstream repository with these changes sans the github actions?

I don't think this particular validation adds value upstream (since it's only about verifying diffs to subtrees are well behaved -- i.e. fairly downstream specific).

Reply:
I'm not sure what I was getting at there with the reference to upstream in my last message so disregard. Obviously this would be a downstream only script.